script.js

```diff
@@ -1,94 +1,99 @@
-// Book constructor
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-// UI constructor
-function UI() {}
+class UI {
+  static addBookToList(book) {
+    const list = document.getElementById('bookList');
+    const li = document.createElement('li');
+    li.innerHTML = `<div class='details'> <div> <strong> <span>"</span>${book.title}<span>"</span> by ${book.author} </strong> </div> <button class='delete'>Remove</button> </div>`;
+    list.appendChild(li);
+  }
 
-UI.prototype.addBookToList = function (book) {
-  const list = document.getElementById('bookList');
-  const p = document.createElement('p');
-  p.innerHTML = `${book.title} <br/> ${book.author} <br/> <button class='delete'>Remove</button> <hr/>`;
-  list.appendChild(p);
-};
+  static clearInputFields() {
+    document.getElementById('titleInput').value = '';
+    document.getElementById('authorInput').value = '';
+  }
 
-UI.prototype.clearInputFields = function () {
-  document.getElementById('titleInput').value = '';
-  document.getElementById('authorInput').value = '';
-};
+  static showAlert(message, className) {
+    const div = document.createElement('div');
+    div.className = `alert ${className}`;
+    div.appendChild(document.createTextNode(message));
+    const container = document.querySelector('body');
+    const form = document.getElementById('bookForm');
+    container.insertBefore(div, form);
 
-UI.prototype.showAlert = function (message, className) {
-  const div = document.createElement('div');
-  div.className = `alert ${className}`;
-  div.appendChild(document.createTextNode(message));
-  const container = document.querySelector('body');
-  const form = document.getElementById('bookForm');
-  container.insertBefore(div, form);
+    setTimeout(() => {
+      document.querySelector('.alert').remove();
+    }, 3000);
+  }
 
-  setTimeout(() => {
-    document.querySelector('.alert').remove();
-  }, 400);
-};
+  static deleteBookFromList(target) {
+    if (target.classList.contains('delete')) {
+      target.parentElement.remove();
+      const title = target.previousElementSibling.textContent;
+      const author = target.previousElementSibling.previousElementSibling.textContent;
+      const book = new Book(title, author);
+      UI.removeFromLocalStorage(book);
+      UI.showAlert('Book removed', 'success');
+    }
```

Comment:
- [ ] Once a book gets deleted by clicking thr remove button it only gets removed from the webpage but does not reflect on the local storage and as such when the page is reloaded previously deleted books are still showing on the webpage. I kindly suggest you make the necessary change to your project so that it functions as it should originally do 👍 ✨

Reply:
This issue is not yet resolved